matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -219,10 +219,10 @@ impl WebRtcSocket {
     /// Note: You have to call [`update_peers`] for this list to be accurate.
     ///
     /// See also: [`WebRtcSocket::disconnected_peers`]
-    pub fn connected_peers(&'_ self) -> impl std::iter::Iterator<Item = &PeerId> {
+    pub fn connected_peers(&'_ self) -> impl std::iter::Iterator<Item = PeerId> + '_ {
```

Comment:
Question: I haven't seen too many examples of Rust anonymous lifetimes like this. Will this clone every PeerId for the iter? I'd hope to avoid that.

Reply:
Yes it will though Uuid is a copy type and should be relatively cheap to clone. Reduced indirection is often good too.

This is why i leaned towards u64, it would definitely be a no-brainer to do a copy since it's the same size as a pointer.